app/js/actions/AuthActions.js

```diff
@@ -40,9 +40,13 @@ export function logoutUser() {
 export function loginUser() {
   return dispatch => {
     dispatch(beginAjaxCall());
-    return webAPI(`/login`, 'POST')
+    return webAPI(`/users`, 'GET')
```

Comment:
please note that there are possibilities of deprecating this `/users` route, if we need the current user, we would create a `/me` route! Moreover `/users` is often the user index, meaning it would return all users